propagator/ottrace/README.md

```diff
@@ -0,0 +1,43 @@
+# opentelemetry-propagator-ottrace
+
+The `opentelemetry-propagator-ottrace` gem contains injectors and extractors for the
+[ottrace context propagation format][ottrace-spec].
+
+## What is OpenTelemetry?
+
+[OpenTelemetry][opentelemetry-home] is an open source observability framework, providing a general-purpose API, SDK, and related tools required for the instrumentation of cloud-native software, frameworks, and libraries.
+
+OpenTelemetry provides a single set of APIs, libraries, agents, and collector services to capture distributed traces and metrics from your application. You can analyze them using Prometheus, Jaeger, and other observability tools.
+
+## How does this gem fit in?
+
+This gem can be used with any OpenTelemetry SDK implementation. This can be the official `opentelemetry-sdk` gem or any other concrete implementation.
+
+## How do I get started?
+
+Install the gem using:
+
+```
+gem install opentelemetry-propagator-ottrace
+```
+
+Or, if you use [bundler][bundler-home], include `opentelemetry-propagator-ottrace` in your `Gemfile`.
+
+## How can I get involved?
+
+The `opentelemetry-propagator-ottrace` gem source is [on github][repo-github], along with related gems including `opentelemetry-api` and `opentelemetry-sdk`.
+
+The OpenTelemetry Ruby gems are maintained by the OpenTelemetry-Ruby special interest group (SIG). You can get involved by joining us in [GitHub Discussions][discussions-url] or attending our weekly meeting. See the [meeting calendar][community-meetings] for dates and times. For more information on this and other language SIGs, see the OpenTelemetry [community page][ruby-sig].
+
+## License
+
+The `opentelemetry-propagator-ottrace` gem is distributed under the Apache 2.0 license. See [LICENSE][license-github] for more information.
+
+[opentelemetry-home]: https://opentelemetry.io
+[bundler-home]: https://bundler.io
+[repo-github]: https://github.com/open-telemetry/opentelemetry-ruby
+[license-github]: https://github.com/open-telemetry/opentelemetry-ruby/blob/main/LICENSE
+[ruby-sig]: https://github.com/open-telemetry/community#ruby-sig
+[community-meetings]: https://github.com/open-telemetry/community#community-meetings
+[discussions-url]: https://github.com/open-telemetry/opentelemetry-ruby/discussions
+[ottrace-spec]: https://github.com/openzipkin/ottrace-propagation
```

Comment:
```suggestion
[ottrace-spec]: https://github.com/opentracing?q=basic&type=&language=
```
Dead link. There's no real canonical doc for this as opentracing doesn't seem to define propagation format explicitly anywhere, the spec( like, here: https://github.com/open-telemetry/opentelemetry-specification/blob/e26d61107e7309aee1e69bb37011043dd6fe6ebe/specification/sdk-environment-variables.md#general-sdk-configuration) seem to reference the above link. 🤷

Reply:
I haven't finished up the docs yet but thanks for the heads up!

Reply:
lgtm